src/libmongoc/tests/test-mongoc-handshake.c

```diff
@@ -276,6 +276,122 @@ test_mongoc_handshake_data_append_success (void)
 }
 
 
+static void
+test_mongoc_handshake_data_append_null_args (void)
+{
+   mock_server_t *server;
+   mongoc_uri_t *uri;
+   mongoc_client_t *client;
+   mongoc_client_pool_t *pool;
+   request_t *request;
+   const bson_t *request_doc;
+   bson_iter_t iter;
+   bson_iter_t md_iter;
+   bson_iter_t inner_iter;
+   const char *val;
+
+   char big_string[HANDSHAKE_MAX_SIZE];
```

Comment:
`big_string` is not used. I suggest removing the declaration and L295-296.

Reply:
Good catch, thank you.